internal/cmdrender/executor.go

```diff
@@ -92,12 +92,18 @@ func (e *Executor) Execute(ctx context.Context) error {
 	at := attribution.Attributor{Resources: hctx.root.resources, CmdGroup: "fn"}
 	at.Process()
 
+	includeMetaResources := true
```

Comment:
will this eventually be changed as a configurable flag? Or kpt always include meta resources in `fn render`

Reply:
Actually, thought some more and realized exposing this as `configurable` flag will make it consistent with other `fn` commands and also provide a way to address the breaking change. Users can run `fn render --include-meta-resources false` to get the previous behavior. I will update the PR.